section-05-production-model-package/tests/test_make_prediction.py

```diff
@@ -0,0 +1,14 @@
+from regression_model.predict import make_prediction
+
+import numpy as np
+
+
+def test_make_prediction(sample_input_data):
+    # When
+    result = make_prediction(input_data=sample_input_data)
+
+    # Then
```

Comment:
I think we should also test that for a certain observation, we obtain the expected value.